arkanoid.c

```diff
@@ -243,6 +245,14 @@ void update_level() {
             }
         }
     }
+    if (is_level_completed()) {
+        level_number++;
+        if (level_number <= MAX_LEVELS) {
+            char filename[20];
+            sprintf(filename, "levels/%d.level", level_number);
+            load_level(filename);
+        }
```

Comment:
A function such as `load_next_level()` in levels.c would add abstraction

Reply:
ok done 👍